Sources/String+Sweet.swift

```diff
@@ -8,4 +8,11 @@ extension String {
 
         return rect.width
     }
+
+    func height(for font: UIFont, constrainedToWidth width: CGFloat) -> CGFloat {
+        let attributes = [NSFontAttributeName: font]
+        let rect = (self as NSString).boundingRect(with: CGSize(width: width, height: CGFloat.greatestFiniteMagnitude), options: .usesLineFragmentOrigin, attributes: attributes, context: nil)
```

Comment:
by word wrapping, character wrapping, etc. This should definitely take that into account. 🤔